webapp/web/js/domjudge.js

```diff
@@ -825,6 +825,20 @@ function enableRefresh($url, $after) {
     }, $after*1000);
     refreshEnabled = true;
     setCookie('domjudge_refresh', 1);
+
+    if(window.location.href == localStorage.getItem('lastUrl')) {
+    	window.scrollTo(0, localStorage.getItem('scrollTop'));
+    } else {
+    	localStorage.setItem('lastUrl', window.location.href);
+    	localStorage.setItem('scrollTop', 42);
+    }
+    var scrollTimeout;
+    document.addEventListener('scroll', function() {
+      clearTimeout(scrollTimeout);
```

Comment:
Isn't doing stuff on scroll going to impact the performance of the browser quite a bit? I don't know, just wondering...

Reply:
nope, it doesn't.